src/assets.rs

```diff
@@ -8,16 +8,14 @@ pub struct LoadingPlugin;
 
 impl Plugin for LoadingPlugin {
     fn build(&self, app: &mut App) {
-        app.add_startup_system(|asset_server: Res<AssetServer>| {
-            asset_server.watch_for_changes().unwrap()
-        });
```

Comment:
Is this enabled by default now or is there a different way to activate this now?

Reply:
If I understood correctly they have to be move to the app creation as default plugins. I have look again.

Reply:
> Is this enabled by default now or is there a different way to activate this now?

I think it is no longer supported: `thread 'main' panicked at 'Watch for changes requires the filesystem_watcher feature and cannot be used on wasm32 / android targets',`

```rust
#[bevy_main]
fn main() {
    App::new()
        .add_plugins(
            DefaultPlugins
                .set(WindowPlugin {
                    window: WindowDescriptor {
                        title: "Whist".to_string(),
                        width: 800.0,
                        height: 600.0,
                        ..default()
                    },
                    ..default()
                })
                .set(AssetPlugin {
                    watch_for_changes: true,
                    ..default()
                }),
        )
        .add_plugin(GamePlugin)
        .run();
}
```

Reply:
Only call it on native then.
Before it only gave a warning.

Reply:
> Only call it on native then. 

How would I do this?